Upstream: Yes.
Source: jT <toyj@union.edu>
Reason:

fixes the symlinks that submakefiles create so that we can have more up2date patches than shitoo.

Edited by Daniel Mierswa <impulze@impulze.org> to match 1.41.8

--- lib/Makefile.elf-lib
+++ lib/Makefile.elf-lib
@@ -48,7 +48,7 @@
 	$(Q) if test "$(ELF_INSTALL_DIR)" = "$(libdir)"; then \
 		$(LN_S) -f $(ELF_SONAME) $(DESTDIR)$(libdir)/$(ELF_IMAGE).so ; \
 	else \
-		$(LN_S) -f $(ELF_INSTALL_DIR)/$(ELF_SONAME) \
+		$(LN_S) -f $(ELF_SONAME) \
 			$(DESTDIR)$(libdir)/$(ELF_IMAGE).so; \
 	fi			
 	$(E) "	LDCONFIG"
